src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,99 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+
+import duke.util.Parser;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.ui.Ui;
+
+import java.io.FileNotFoundException;
+
+/**
+ * The main class of CS2103 iP.
+ * It is the main body of this task tracking software
+ * that drives the main logic.
+ */
+public class Duke {
+    private static final String DEFAULT_PATH = "./data/duke.txt";
+    private Ui ui;
+    private Storage storage;
+    private TaskList tasks;
+
+    public static void main(String[] args) {
+        Duke duke = new Duke(DEFAULT_PATH);
+        duke.run();
+    }
+
+    /**
+     * Main logic of Duke,
```

Comment:
Consider starting the sentence with a verb like `Starts` to follow the [Java Coding Standard](https://se-education.org/guides/conventions/java/intermediate.html)